crates/bevy_ecs/examples/change_detection.rs

```diff
@@ -6,6 +6,8 @@
 //! To demonstrate change detection, there are some console outputs based on changes in
 //! the `EntityCounter` resource and updated Age components
 
+#![expect(clippy::std_instead_of_core)]
```

Comment:
These examples require this `expect` attribute because they are inside the `bevy_ecs` crate and thus subject to the workspace lints. It's not ideal, but there are only a couple of cases where this is required.